Makefile.am

```diff
@@ -3,11 +3,13 @@ dist_bin_SCRIPTS = \
 	create-update.sh \
 	build-chroots.sh \
 	init-mix.sh \
-	init-versions.sh \
```

Comment:
No longer needed. The builder.conf hosts the values, and I will adjust the scripts to auto-increment or set the next mix value when starting to build the version. Imho we shouldn't need a whole file to edit a version #.